command/token/helper_internal.go

```diff
@@ -21,11 +21,16 @@ type InternalTokenHelper struct {
 // populateTokenPath figures out the token path using homedir to get the user's
 // home directory
 func (i *InternalTokenHelper) populateTokenPath() {
-	homePath, err := homedir.Dir()
-	if err != nil {
-		panic(fmt.Sprintf("error getting user's home directory: %v", err))
+        if runtime.GOOS == "windows" {
+		pathSeparator := "\\"
+	} else {
+		pathSeparator := "/"
 	}
-	i.tokenPath = homePath + "/.vault-token"
+        homePath, err := homedir.Dir()
+        if err != nil {
+                panic(fmt.Sprintf("error getting user's home directory: %v", err))
+        }
+        i.tokenPath = homePath + pathSeparator + ".vault-token"
```

Comment:
I _think_ `filepath.Join()` would take care of this in a cross platform way?

Reply:
Yep, good call.